packages/microcode/microcode.spec

```diff
@@ -110,6 +107,7 @@ install -p -m 0644 intel/intel-ucode/* %{buildroot}%{_cross_libdir}/firmware/int
 %dir %{_cross_libdir}/firmware
 %dir %{_cross_libdir}/firmware/intel-ucode
 %{_cross_libdir}/firmware/intel-ucode/??-??-??
+%{_cross_libdir}/firmware/intel-ucode/??-??-??_DUPLICATE
```

Comment:
Those made me curious. According to [this](https://github.com/intel/Intel-Linux-Processor-Microcode-Data-Files/issues/68) and [this](https://github.com/intel/Intel-Linux-Processor-Microcode-Data-Files/issues/69) issue, they were (accidental?) commits of a previous microcode revision and to be ignored. We can do that by adding an `%exclude`.